src/Components/Sidebar/index.jsx

```
@@ -1,862 +1,862 @@
+		// If the user is NOT a superadmin, remove the "Team" option
+		if (user.role && !user.role.includes("superadmin")) {
+			filteredAccountMenuItems = filteredAccountMenuItems.filter(
+				(item) => item.name !== "Team"
+			);
+		}
+
+		return filteredAccountMenuItems.map((item) => (
+			<MenuItem
+				key={item.name}
+				onClick={() => {
+				     closePopup()
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)